compiler/src/iree/compiler/Codegen/Common/CPU/CPUMaterializeEncodingPass.cpp

```diff
@@ -161,13 +161,24 @@ enumerateMatmulTileX86_64(EncodingUser user, TypeRange elementTypes,
     // reconsider when taking advantage of native f16/bf16 arithmetic when the
     // accumulator itself is f16/bf16.
     if (hasFeature(target, "+avx512f")) {
-      return {
-          TileMxNxK{16, 16, 1}, // Aim to use VFMADD* (zmm).
-          TileMxNxK{8, 16, 1},  // Truncation of the above.
-          TileMxNxK{4, 16, 1},  // Truncation of the above.
-          TileMxNxK{2, 16, 1},  // Truncation of the above.
-          TileMxNxK{1, 16, 1},  // Truncation of the above.
-      };
+      if (hasUkernel(target)) {
+        return {
+            TileMxNxK{16, 16, 1}, // Aim to use VFMADD* (zmm).
+            TileMxNxK{8, 16, 1},  // Truncation of the above.
+            TileMxNxK{4, 16, 1},  // Truncation of the above.
+            TileMxNxK{2, 16, 1},  // Truncation of the above.
+            TileMxNxK{1, 16, 1},  // Truncation of the above.
+        };
+      } else {
+        // Code generation tile sizes.
+        return {
+            TileMxNxK{16, 16, 1}, // Aim to use VFMADD* (zmm).
+            TileMxNxK{8, 32, 1},  // Use same number of accumulators.
+            TileMxNxK{4, 64, 1},  // Use same number of accumulators.
+            TileMxNxK{2, 64, 1},  // Use half the number of accumulators.
+            TileMxNxK{1, 128, 1}, // Use half the number of accumulators.
+        };
```

Comment:
It looks like the comment you added is gone after you rebase it?

Reply:
I''m talking about the review comment before your force-push: https://github.com/openxla/iree/pull/15421#discussion_r1396448191

Reply:
Not sure but I addressed the feedback provided.

Reply:
I think it was not just `// Code generation tile sizes.` before force-push.. You added a comment about "why they have different tile sizes comparing to ukernels.", but it does not show up after force-push..